src/index.js

```diff
@@ -60,40 +60,39 @@ const displaySchedules = () => {
   const myImage = new Image();
   // eslint-disable-next-line camelcase
   myImage.src = noImage;
-  const moviesContainer = document.getElementById("movies-container");
-  let container = "";
+  const moviesContainer = document.getElementById('movies-container');
+  let container = '';
 
   schedules.then((schedule) => {
-    let titlesArray = [];
+    const titlesArray = [];
 
     schedule.forEach((item) => {
       // eslint-disable-next-line no-underscore-dangle
```

Comment:
Let's remove all disabled Linters, Linting makes your code look cleaner and professional 🙂 

Kindly check for other occurrences 👍